lib/generators/hyrax/templates/config/hyrax.rb

```diff
@@ -74,6 +74,10 @@
   # Path to the file derivatives creation tool
   # config.libreoffice_path = "soffice"
 
+  # Option to enable/disable full text extraction from PDFs
+  # Default is false, set to true to skip extraction
```

Comment:
Agreed. See comment above. Was having weird testing and felt odd about the naming vs. the true/false value. I'll update this and rework it in the AM. ;-)